pep-9999.rst

```diff
@@ -0,0 +1,109 @@
+PEP: 9999
+Title: Disabling bdist_egg distribution uploads on PyPI
+Author: William Woodruff <william@yossarian.net>
+Sponsor: Donald Stufft <donald@stufft.io>
+PEP-Delegate: Donald Stufft <donald@stufft.io>
+Status: Draft
+Type: Process
+Topic: Packaging
+Content-Type: text/x-rst
+Created: 06-Jun-2023
+Post-History:
+Resolution:
+
+Abstract
+========
+
+This PEP recommends deprecating and then disabling new uploads of the
+``bdist_egg`` distribution type on PyPI. In a parallel move, this PEP recommends
+deprecating and then disabling new uploads of distribution filenames that have
+the ``.egg`` suffix.
+
+It does not recommend removing or otherwise affecting any previously
+uploaded ``bdist_egg`` distributions.
+
+Rationale
+=========
+
+Previous Work
+-------------
+
+The groundwork for this proposal was established with :pep:`527`, which
+proposed deprecating and eventually removing upload support for a handful
+of un(der)used file extensions and distribution types.
+
+In particular, :pep:`527` proposed the removal of the ``bdist_dumb``,
+``bdist_rpm``, ``bdist_dmg``, ``bdist_msi``, and ``bdist_wininst`` distribution
+types, as well as the ``.tar``, ``.tar.bz2``, ``.tar.xz``, ``.tar.Z``,
+``.tgz``, and ``.tbz`` file extensions on distribution filenames.
+
+:pep:`527` was fully enacted with
+`PR #7529 <https://github.com/pypi/warehouse/pull/7529>`_ to Warehouse,
+which was merged on 13 April 2020.
+
+The ``bdist_egg`` format
+------------------------
+
+The ``bdist_egg`` filetype identifies distributions in the
+`Egg format <https://packaging.python.org/en/latest/glossary/#term-Egg>`_. The
```

Comment:
```suggestion
:term:`Egg format <packaging:Egg>`. The
```

No need for a hardcoded link here—since we have `packaging.python.org` as an intersphinx target, you can crossreference the glossary term directly.